sdk/storage/Azure.Storage.Files.DataLake/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 - Added support for service version 2021-06-08.
 - Added support for Customer Provided Key server-side encryption of files.
 - Added ability to retrieve path CreatedOn and ExpiresOn times with DataLakeFileSystemClient.GetPaths() and .GetPathsAsync().
+- Made DataLakeFileSystemClient.GetPathClient() public.
```

Comment:
nitpick: I would just say "added support for <api>" here for consistent messaging.

Reply:
Fixed.